deploy.py

```diff
@@ -205,6 +196,17 @@ def assert_python_library(module):
            f"Try to run: pip3 install {module}"))
 
 
+def create_metadata(firmware_image, partition_address):
+  timestamp = struct.pack("<I", int(time.time()))
```

Comment:
To avoid issues with timezones, maybe it's better to use UTC and therefore `datetime` instead of `time`.
`int(datetime.datetime.utcnow().timestamp())`